test/e2e/selectors/selectors_test.go

```diff
@@ -0,0 +1,144 @@
+package field_selectors
```

Comment:
update package name according to the folder name
```suggestion
package selectors
```